generate_changelog.sh

```
@@ -1,31 +1,256 @@
+check_command grep ;
+check_command head ;
+check_command sed ;
+check_command sort ;
+check_command tail ;
+check_command tee ;
+check_command tr ;
+check_command uniq ;
+check_command xargs ;
+
+# rest of the script vars
+LOG_FILE="chglog_generation_${PPID}.log"
```

Shellcheck (0.10.0)

[warning] LOG_FILE appears unused. Verify use (or export if used externally).

(SC2034)